.github/workflows/release_extension.yaml

```diff
@@ -0,0 +1,51 @@
+name: Release VSCode Extension
+on:
+  push:
+    tags:
+      - 'vscode@v*'
```

Comment:
You need to either change the prefix here or in the CircleCI config: https://github.com/TobikoData/sqlmesh/blob/main/.circleci/config.yml#L17. Since both tags happen to start with `v`